betty/project/__init__.py

```diff
@@ -379,7 +379,7 @@ async def _init_extensions(self) -> ProjectExtensions:
         if theme_count == 0:
             logging.getLogger().warning(
                 _(
-                    'Your project has no theme enabled. This means your site\'s pages may look bare. Try the "cotton-candy" extension.'
+                    'Your project has no theme enabled. This means your site\'s pages may look bare. Try the "raspberry-mint" extension.'
```

Comment:
Show this in the default page base template as well. This should address issues that users have reported as well, where their sites are bare.